src/ralph/util/demo/devices.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+# from ralph.discovery.models import DeviceType
+from ralph.cmdb.models_ci import CIType
+
+from ralph_assets.tests.utils.assets import (
+    DataCenterFactory,
+    RackFactory,
+    ServerRoomFactory,
+)
+from ralph.discovery.tests.util import (
+    DeprecatedDataCenterFactory,
+    DeprecatedRackFactory,
+    DeviceFactory,
+    DiscoveryQueueFactory,
+)
+from ralph.cmdb.tests.utils import (
+    CIRelationFactory,
+    DeviceEnvironmentFactory,
+    ServiceCatalogFactory,
+)
+from ralph.cmdb.models import (
+    # CIRelation,
+    CI_RELATION_TYPES,
+)
+
+from ralph.util.demo import DemoData, register
+
+
+@register
+class DemoDeviceEnvironment(DemoData):
+    name = 'envs'
+    title = 'Environments'
+
+    def generate_data(self, data):
+        # CITypes loaded from fixtures
+        ci_type = CIType.objects.get(id=11)
+        return {
+            'prod': DeviceEnvironmentFactory.create(
```

Comment:
btw what do you think about such a construction - wouldn't it be better?
```
envs = [('prod', 'Prod'), ('test', 'Test'), ('dev', 'Dev')]
return {slug: DeviceEnvironmentFactory(name=name, type=ci_type) for (slug, name) in envs.iteritems()}
```
eventually you could add some parser from name to slug (ex. `name.replace(' ', '_').lower()` in basic version) and just define list of names.